tool/tsh/tsh.go

```diff
@@ -1506,6 +1508,12 @@ func onLogin(cf *CLIConf) error {
 		}
 	}
 
+	pingResp, err := tc.Ping(cf.Context)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	tc.LoadAllCAs = pingResp.Auth.LoadAllHostCAs
```

Comment:
This should move into [applyProxySettings](https://github.com/gravitational/teleport/blob/f1b646eb35ad76648ae034472d29bc33da9adcea/lib/client/api.go#L3956).